android/app/src/main/kotlin/com/fariszr/dualmate/flutter/AndroidScheduleTodayWidget.kt

```
@@ -6,22 +6,28 @@ import android.content.Intent
 import io.flutter.plugin.common.MethodCall
 import io.flutter.plugin.common.MethodChannel
 
 
-class AndroidScheduleTodayWidget(private val context: Context) : MethodChannel.MethodCallHandler {
-    fun setupMethodChannel(@NonNull flutterEngine: FlutterEngine) {
-        MethodChannel(
-                flutterEngine.dartExecutor.binaryMessenger,
-                "com.fariszr.dualmate/widget")
-                .setMethodCallHandler(this)
+class AndroidScheduleTodayWidget : FlutterPlugin, MethodChannel.MethodCallHandler {
+    private lateinit var context: Context
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)